packages/reassure-cli/src/commands/measure.ts

```diff
@@ -40,10 +43,17 @@ export async function run(options: MeasureOptions) {
   const header = { metadata };
   writeFileSync(outputFile, JSON.stringify(header) + '\n');
 
-  const testRunnerPath = process.env.TEST_RUNNER_PATH ?? getJestBinPath();
+  if (!RUNNERS.includes(options.runner)) {
+    logger.error(`❌ Invalid runner provided: ${options.runner}. Valid options are 'jest' and 'vitest'`);
```

Comment:
nit: could enumerate the options from your `RUNNERS` const. The native node test runner could be added later for example